packages/dashboard/src/pages/Wallets/CreateWallet.tsx

```diff
@@ -0,0 +1,141 @@
+import { Button, createStyles, experimentalStyled as styled, FilledInput, Tab, makeStyles } from '@material-ui/core'
+import { ButtonGroupTabList, MaskColorVar } from '@dimensiondev/maskbook-theme'
+import { memo, useMemo, useState } from 'react'
+import { TabContext, TabPanel } from '@material-ui/lab'
+import { RefreshIcon } from '@dimensiondev/icons'
+import { MnemonicReveal } from '../../components/Mnemonic'
+import { MaskAlert } from '../../components/MaskAlert'
+import { useDashboardI18N } from '../../locales'
+
+const Container = styled('div')`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`
+
+// TODO: mobile
+const ButtonGroupTabContainer = styled('div')`
+    width: 582px;
+`
+
+const Refresh = styled('div')(
+    ({ theme }) => `
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    /* TODO: Mobile */
+    width: 584px;
+    margin: ${theme.spacing(2, 0)};
+    font-size: ${theme.typography.fontSize};
+    line-height: 20px;
+    color: ${theme.palette.primary.main};
+    cursor: pointer;
+`,
+)
+
+const MnemonicGeneratorContainer = styled('div')(
+    ({ theme }) => `
+    padding: ${theme.spacing(4, 5)};
+    background-color: ${theme.palette.background.default};
+    border-radius: 8px;
+`,
+)
+
+const ControlContainer = styled('div')(
+    ({ theme }) => `
+    margin-top: ${theme.spacing(6)};
+    display: grid;
+    justify-content: center;
+    grid-template-columns: repeat(2, 180px);
+    gap: 24px;
+    /* TODO: Mobile */
+    width: 584px;
+`,
+)
+
+const AlertContainer = styled('div')(
+    ({ theme }) => `
+    /* TODO: Mobile */
+    width: 676px;
+    margin-top: ${theme.spacing(7)};
+    color: ${MaskColorVar.textSecondary};
+`,
+)
+
+const PrivateKeyInput = styled(FilledInput)(
+    ({ theme }) => `
+    /* TODO: Mobile */
+    width: 582px;
+    height: 182px;
+    margin-top: ${theme.spacing(3)};
+`,
+)
+
+const useTabPanelStyles = makeStyles(() =>
+    createStyles({
+        root: {
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            padding: 0,
+        },
+    }),
+)
+
+export const CreateWallet = memo(() => {
+    const tabClasses = useTabPanelStyles()
+    const t = useDashboardI18N()
+
+    const walletTabs = useMemo(
```

Comment:
Also wrong. The correct way is to split value(what stored in the state) and label (what displayed to the user)